commune/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for commune project.
+
+Generated by 'django-admin startproject' using Django 4.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-57^uz74h_web1fi==4!c+=8s2bc8t$cr!rwkc=80vsl8@no%5d'
```

Comment:
we should probably put this key in a config file along with the config file I'm planning for Flake8

Reply:
we can do it later as improvement